lib/quantum/normalizer.ex

```diff
@@ -110,6 +110,7 @@ defmodule Quantum.Normalizer do
       schedule: extract(:schedule, opts),
       task: extract(:task, opts),
       args: extract(:args, opts, []),
+      overlap: extract(:overlap, opts, Application.get_env(:quantum, :default_overlap, true)),
```

Comment:
You are right. It was just a quick fix to resolve this issue.
There is a more elegant way which I plan to submit on the weekend.